terraform/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+}
+
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = "latest/stable"
+}
```

Comment:
The TF style guide suggests this file keeps variables in alphabetical order (ref https://developer.hashicorp.com/terraform/language/style#file-names)